README.md

```diff
@@ -2048,6 +2053,90 @@ Probe type. Valid values are 'icmp', 'dns', 'tcp', 'udp' or keyword 'default'.
 Weight for traffic distribution. Valid values are Integer, keyword 'default'.
 
 --
+### Type: cisco_itd_service
+
+Manages configuration of ITD (Intelligent Traffic Director) service
+
+| Platform | OS Minimum Version | Module Minimum Version |
+|----------|:------------------:|:----------------------:|
+| N9k      | 7.0(3)I3(1)        | 1.3.0                  |
+| N30xx    | unsupported        | unsupported            |
+| N31xx    | unsupported        | unsupported            |
+| N56xx    | unsupported        | unsupported            |
+| N6k      | unsupported        | unsupported            |
+| N7k      | 7.3(0)D1(1)        | 1.3.0                  |
+| N8k      | unsupported        | unsupported            |
+| IOS XR   | unsupported        | unsupported            |
+
+#### <a name="cisco_itd_service-caveats">Caveats</a>
+
+| Property | Caveat Description |
+|:--------|:-------------|
+| `nat_destination` | Supported only on N7k |
+| `peer_local` | Supported only on N9k |
+| `peer_vdc` | Supported only on N7k |
+
+#### Parameters
+
+##### `ensure`
+Determine whether the config should be present or not. Valid values are 'present' and 'absent'.
+
+##### `access_list`
+User ACL name. Valid values are String or 'default'.
+
+##### `device_group`
+Device group name where this service belongs. Valid values are String or 'default'.
+
+##### `exclude_access_list`
+User ACL name to exclude from redirection. Valid values are String or 'default'.
+
+##### `fail_action`
+Failaction node reassign. Valid values are true, false or 'default'.
+
+##### `ingress_interface`
+Ingress interface. Valid values are an array of [interface, next-hop] pairs or 'default'.
+
+##### `load_bal_enable`
+Enable or disable load balance. Valid values are true, false or 'default'.
+
+##### `load_bal_buckets`
+Buckets for traffic distribution (in powers of 2). Valid values are Integer, keyword 'default'.
+
+##### `load_bal_mask_pos`
```

Comment:
This is coded as load_bal_mask_pos all over including the puppet-agent. If you want me to change in the README.md file, it is ok, otherwise, we have to change everywhere including the puppet-agent which will need another PR. Can you tell me what you want me to do?